src/styles/_header.scss

```diff
@@ -37,5 +37,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition-duration: 300ms;
```

Comment:
same

Reply:
Successfully deployed to gh-pages!

- [DEMO LINK](https://vladyslava-buzova.github.io/layout_catalog_hovers/)